webpack/assets/javascripts/react_app/components/common/DateTimePicker/DatePicker.js

```diff
@@ -77,10 +77,12 @@ DatePicker.propTypes = {
   value: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
   locale: PropTypes.string,
   weekStartsOn: PropTypes.number,
+  popoverPlacement: PropTypes.string,
```

Comment:
can you rename to `placement` like in the DateTimePicker?
https://github.com/theforeman/foreman/blob/fc48ba39fabc088f3acccc94066b638cb595a8c6/webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.js#L126